docker/pipeline_dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.10
+
+WORKDIR /pipeline
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+RUN pip install python-dotenv
+
+CMD ["python", "etl.py"]
```

Comment:
Is this related to the pull request? We already have a dockerfile for this in pipline (and this needs to be a lambda dockerfile)